pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3683,10 +3684,10 @@ int uprobe_syscall_trigger(struct pt_regs *ctx)
 
     #if defined(bpf_target_x86)
         // go1.17, go1.18, go 1.19
-        magic_num = ctx->bx;                                          // 1st arg
```

Comment:
@AsafEitani do you remember if there was a reason to put it here?

Reply:
No need, we just need it to stack based argument passing